BSP/Simulator/Src/BSP_PWM.c

```diff
@@ -8,6 +8,16 @@
 
 static bool initialized = false;
 static uint32_t pinSpeeds[] = {0, 0, 0, 0, 0};
+static bool killSwitchLockdown = false;
+
+/**
+ * @brief Only used for simulator purposes.
+ * 
+ * @param val to set kill switch to
+ */
+void BSP_PWM_SetKillSwitch(bool val) {
```

Comment:
There shouldn't be another function in the code that sets the kill switch. The BSP should be the same in simulator and STM